core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review.component.html

```diff
@@ -1,8 +1,8 @@
 <div class="modal-header e2e-test-question-suggestion-review-modal-header">
   <div>
-    <h3 class="oppia-question-header" tabindex="0">
+    <h1 class="oppia-question-header" tabindex="0">
```

Comment:
Will it change the size of the text significantly? Also, do we need to make a similar change for the translation suggestion review model component?

Reply:
Hi @Patel-Muhammad, can you point to the changes made here?

Reply:
@Lawful2002 PTAL at this:
Fixes this issue- #17182 
![Screenshot 2023-06-20 110317](https://github.com/oppia/oppia/assets/96219910/0871bce2-2824-4625-b2ea-e285ca170052)
![Screenshot 2023-06-20 115614](https://github.com/oppia/oppia/assets/96219910/4e490a4b-c1a0-4c09-b1a5-905438255486)